### PR TITLE
chore: Update to meteor 2.13

### PR DESCRIPTION
**Pull request by jstarpl**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Meteor framework update

* **What is the current behavior?** (You can also link to an open issue here)

Uses meteor 2.12

* **What is the new behavior (if this is a feature change)?**

Uses meteor 2.13 and new Node 14 ESM by Meteor

* **Other information**:

**Status**

- [ ] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [ ] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/.meteor/packages [hunk 1]
```diff
@@ -10,27 +10,27 @@
...
 
-standard-minifier-css@1.8.1 # CSS minifier run for production mode
-standard-minifier-js@2.8.0 # JS minifier run for production mode
+standard-minifier-css@1.9.2 # CSS minifier run for production mode
+standard-minifier-js@2.8.1 # JS minifier run for production mode
 es5-shim@4.8.0 # ECMAScript 5 compatibility for older browsers
-ecmascript@0.16.2 # Enable ECMAScript2015+ syntax in app code
-typescript@4.5.4 # Enable TypeScript syntax in .ts and .tsx modules
+ecmascript@0.16.7 # Enable ECMAScript2015+ syntax in app code
+typescript@4.9.4 # Enable TypeScript syntax in .ts and .tsx modules
```
Comment (Julusian) [inline meteor/.meteor/packages:20]: The version of typescript used in the `package.json` should be updated to match

File: meteor/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:experimental
 # BUILD IMAGE
-FROM node:14
+FROM meteor/node:14.21.4
 RUN curl "https://install.meteor.com/?release=2.12" | sh
```
Comment (Julusian) [inline meteor/Dockerfile:4]:
```suggestion
RUN curl "https://install.meteor.com/?release=2.13.3" | sh
```

Comment (Julusian) [reply meteor/Dockerfile:4]: There is another line like this in `.github/actions/` somewhere, which should be updated too

**Review by Julusian — APPROVED** [review]

File: meteor/Dockerfile (hunk 2 above)
Comment (jstarpl) [reply meteor/Dockerfile]: Thanks, found it

**Review by nytamin — APPROVED** [review]

**Pull request merged**